File: src/main/java/com/picpay/rh/entity/Funcionario.java
package com.picpay.rh.entity;

import java.io.Serializable;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter @Setter @NoArgsConstructor @AllArgsConstructor
public class Funcionario implements Serializable {

    private Long id;

    private String nome;

    private String email;

    private String telefone;

    private String cargo;

    private String departamento;

    private double salario;

    private String cidade;

    private String status;
    
}

File: frontend/src/models/funcionario.ts
export interface FuncionarioRequestDTO {
	nome: string
	email: string
	telefone: string
	cargo: string
	departamento: string
	salario: number
	cidade: string
	status: string
}

export interface FuncionarioResponseDTO extends FuncionarioRequestDTO {
	id: number
}
